libs/utils/utils.js

```diff
@@ -448,6 +448,7 @@ export async function loadBlock(block) {
   }
 
   const name = block.classList[0];
+  const voidStyles = block.classList.contains('void-styles');
```

Comment:
Do you have a scenario in the real world where this would be used?

Reply:
Not by GWP, no. The blocks not requiring styles are auto blocks, so they get an additional class that hints that styles shouldn't be loaded. Another way to do this is have a check here on whether the current block is an auto block and then see if styles are needed. Do you prefer the latter or have an even better idea?